### ISSUE TITLE
Process and Process Runtime Resources: typos in references to Java system properties

### ISSUE THREAD

**Issue by ghannington-Rocket** [open-telemetry/semantic-conventions#1395]
## Excerpt of content

From the [Markdown source](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/process.md?plain=1#L152-L157) of the topic "[Process and Process Runtime Resources](https://opentelemetry.io/docs/specs/semconv/resource/process/#java-runtimes)":

```markdown
Java instrumentation SHOULD fill in the values by copying from system properties.

- `process.runtime.name` - Fill in the value of `java.runtime.name` as is
- `process.runtime.version` - Fill in the value of `java.runtime.version` as is
- `process.runtime.description` - Fill in the values of `java.vm.vendor`, `java.vm.name`, `java.vm.version`
  in that order, separated by spaces.
```
## Issue

The qualifier `runtime` in `java.runtime.name` and `java.runtime.version` is a typo. There are no such Java system properties.

## Suggestion

In those particular references, replace the `runtime` qualifier with `vm`.

## Discussion

I think the docs are referring to the Java system properties `java.vm.name` and `java.vm.version`. I suspect that the writer has muddled `process.runtime.*` OpenTelemetry attributes with `java.vm.*` Java system properties.

## References

- Oracle docs: [Java 17 system properties](https://docs.oracle.com/en/java/javase/17/docs/api/system-properties.html)
- Related Stack Overflow question (mentions various "version"-related Java system properties, with examples): "[How to find the JVM version from a program](https://stackoverflow.com/questions/5103121/how-to-find-the-jvm-version-from-a-program)?"

**Comment by joaopgrassi** [comment]
Hi @ghannington-Rocket 

I just tried and for me the keys actually are working as expected:

```java
String runtime = java.lang.System.getProperty("java.runtime.name");
String runtimeVersion = java.lang.System.getProperty("java.runtime.version");
```

The result is:
![image](https://github.com/user-attachments/assets/4be8b217-a8ec-42cc-868a-50a3e778954e)

Did you tried something that didn't work?

**Comment by ghannington-Rocket** [comment]
I'm wrong. Sincere apologies for wasting your time.

I should have tried this myself, but didn't. I'm a Java noob. I relied on the Oracle docs topic that I cited, [System Properties](https://docs.oracle.com/en/java/javase/17/docs/api/system-properties.html), to be comprehensive. I'm guessing there are technical reasons why `java.runtime.name` and `java.runtime.version` don't appear in that topic, but _are_ returned by `System.getProperties()`. I'm on a learning curve.

Thanks for correcting me gently!
